src/beam/contrib/auth/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import include, path
+
+from beam.contrib.auth.views import GroupViewSet, UserViewSet
+
+urlpatterns = [
```

Comment:
hmm, `django.contrib.auth` includes urls for login, logout and password reset too, doesn't it?

Reply:
Yeah but my thought was that we don't want to force anyone to use the default urls just because they want to use the viewsets. This way you can just include both if you want both.